### PR TITLE
client.ar.yml

### PR DESCRIPTION
**Pull request by Raddah**
for Arabic translation

### CODE REVIEW

File: config/locales/client.ar.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+en:
```
Comment (ZogStriP) [inline config/locales/client.ar.yml]: This line has to match the locale.

File: config/locales/client.en.yml [hunk 2]
```diff
@@ -1,11 +0,0 @@
-en:
```
Comment (ZogStriP) [inline config/locales/client.en.yml]: Why did you remove the `en` locale?

Comment (Raddah) [inline config/locales/client.en.yml]: I didn't remove it >>> may be by mistake I thought I already fork it  :) 

sorry if that happen

Comment (Raddah) [inline config/locales/client.en.yml]: sorry by mistake I didn't mean it
